pdk/1.0/modules/ROOT/pages/policies-pdk-create-schema-definition.adoc

```diff
@@ -0,0 +1,442 @@
++
+[source,yaml]
+----
+properties:
+    squareRoot:
+        type: number
+        default: 1.414213562
+    seconds:
+        type: number
+        default: 10
+----
++
+Integer::
++
+The `integer` type specifies data as an integer value.
++
+For example:
++
+[source,yaml]
+----
+properties:
+    maxHeaders:
+        type: integer
+        default: 10
+----
++
+Boolean::
++
+The `boolean` type specifies data as a boolean value. Supported values are `true` or `false`.
+For example:
++
+[source,yaml]
+----
+properties:
+    skipValidation:
+        type: boolean
+        default: false
+    requireCredentials:
+        type: boolean
+        default: true
+----
++
+Array::
++
+The `array` type specifies the data as a list of ordered elements. For the `array` type, you must also specify `items.type` to specify the data type of the items in the array.
++
+For example:
++
+[source,yaml]
+----
+properties:
+    scopes:
+        type: array
+        items:
+            type: string
+        default: ["READ", "WRITE"]
+----
++
+Object::
++
+The `object` type specifies the data as a list of ordered elements with multiple properties. For the `object` type, you must also specify `properties.type` for each element in the object.
++
+For example:
++
+[source,yaml]
+----
+properties:
+    user:
+      type: object
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+      required:
+        - id
+        - name
+      default: {"id": "1", "name": "John Doe"}
+----
+
+[[dataweave-expressions]]
+=== DataWeave Expressions
+
+The `dataweave` format for `string` type parameters specifies the input parameter as a DataWeave expression.
+
+The `dataweave` variable supports the following expression bindings:
+
+[source,yaml]
+----
+<configuration-parameter>:
+ type: string
+ format: dataweave
+ bindings:
+   payload:
+     mimeType: text
+   attributes: true
+   authentication: false
+   vars: []
+----
+
+* `payload`: The body of the incoming request. Define the `mimeType` as either `text` or `json` to set how to read the payload. By default, `mimeType` is set to `text`. To restrict the use of the `payload` variable, you must exclude the payload variable, for example:
++
+[source,yaml]
+----
+bindings:
+   attributes: true
+   authentication: false
+   vars: []
+----
+
+* `attributes`: Attributes, such as headers, of the incoming request. Set `attributes` to `true` to enable the policy to read the attributes of the incoming request. By default, `attributes` is set to `true`.
+* `authentication`: Authentication information of the incoming request. Set `authentication` to `true` to enable the policy to read authentication details of the incoming request. By default, `authentication` is set to `false`.
+* `vars`: Specifies the names of the variables that are available for the policy to use. By default, `vars` is empty. If the user inputs a variable that is not defined in the schema, the policy evaluates the variable as `null`.
```

Comment:
```suggestion
* `vars`: Specifies the names of the variables that are available for the policy to use. By default, `vars` is empty. If the user inputs a variable that is not defined in the schema, the policy evaluates the variable as `null`.
```
Who is "the user" here? The developer?

Reply:
https://github.com/mulesoft/docs-gateway/pull/504